drift-client/src/test/java/io/airlift/drift/client/TestDriftMethodInvocation.java

```diff
@@ -622,13 +622,13 @@ private static void assertClassifiedException(Throwable cause, ExceptionClassifi
         assertTrue(cause instanceof ClassifiedException);
         ClassifiedException classifiedException = (ClassifiedException) cause;
         assertEquals(classifiedException.getClassification(), exceptionClassification);
-        assertRetriesFailedInformation(classifiedException, expectedRetries + 1, expectedRetries + 1, 0);
+        assertRetriesFailedInformation(classifiedException, 0, expectedRetries + 1, 0);
     }
 
-    private static void assertRetriesFailedInformation(Throwable exception, int expectedConnectionAttempts, int expectedInvocationAttempts, int expectedOverloaded)
+    private static void assertRetriesFailedInformation(Throwable exception, int expectedFailedConnections, int expectedInvocationAttempts, int expectedOverloaded)
     {
         RetriesFailedException retriesFailedException = getRetriesFailedException(exception);
-        assertEquals(retriesFailedException.getConnectionAttempts(), expectedConnectionAttempts);
+        assertEquals(retriesFailedException.getFailedConnections(), expectedFailedConnections);
```

Comment:
Looks like expected is now always zero. Can we add a test that has a connection failure?